downstream/modules/aap-on-azure/proc-peer-vwan-hub-to-aap.adoc

```diff
@@ -0,0 +1,33 @@
+[id="proc-peer-vwan-hub-to-aap"]
+
+= Peering a VWAN Hub to the {AAPonAzureNameShort} Network
+
+Before you can peer a VWAN with the {AAPonAzureNameShort} VNET, you must have connected a hub network and at least one spoke network to the hub network of the Azure VWAN that you intend to connect {AAPonAzureNameShort} to.
+
+.Prerequisites
+
+* A pre-configured Azure VWAN that is already tested and working.
+* One or more of the following connections to the VWAN:
+** A DMZ network that contains Azure virtual machines that users will use to access {AAPonAzureNameShort}.
+** A DMZ network that contains an Azure virtual machine that can be used with SSH tunneling to access {AAPonAzureNameShort}.
+** A VPN or Direct Connect service to your local network that routes traffic from machines on your local network to {AAPonAzureNameShort}.
+
+Peering the {AAPonAzureNameShort} VNET to your VWAN follows the same approach as peering any other VNET or network with a VWAN.
```

Comment:
I'm not sure this sentence is that informative or useful. Seems like fluff to me.